doc/api/crypto.md

```diff
@@ -2085,10 +2086,27 @@ the corresponding digest algorithm. This does not work for all signature
 algorithms, such as `'ecdsa-with-SHA256'`, so it is best to always use digest
 algorithm names.
 
+### `crypto.diffieHellman(ourKey, theirKey)`
```

Comment:
The key order here is going to be impossible to remember, `diffieHellman({private: keyObj, public: keyObj})` would be possible, but verbose and out of character with the rest of the API. How about just allow any order, as long as one is private, and one is public?

Reply:
my ephemeral private key, their public key is what i’d be looking for as a consumer

Reply:
I’m a fan of an options object for verbose naming :+1:

Reply:
How about `{ publicKey, privateKey }`? Would you prefer that over the current API @sam-github @addaleax?